prestoadmin/server.py

```diff
@@ -119,7 +121,7 @@ def _check_rpm_uncorrupted(rpm_path):
 
     def _check_if_absolute_path(self):
         if os.path.isfile(self.local_path) and \
-           self._check_rpm_uncorrupted(self.local_path):
+                self._check_rpm_uncorrupted(self.local_path):
```

Comment:
You can probably get rid of the line splice. I think we increased the line length to be 120 chars.